device/Microsoft.Azure.Devices.Client/DeviceClient.cs

```diff
@@ -245,7 +246,7 @@ public void SetRetryPolicy(IRetryPolicy retryPolicy)
         /// </summary>
         Object twinPatchCallbackContext = null;
 
-        internal int CurrentMessageCount = 0;
+        private int CurrentMessageCount = 0;
```

Comment:
Nit: since it's a private and not a const it should be `_currentMessageCount`.

Reply:
Thanks a lot for your help about the code review, I will change it!